xml/gl.xml

```diff
@@ -45806,6 +45808,11 @@ typedef unsigned int GLhandleARB;
             </require>
         </extension>
         <extension name="GL_NVX_blend_equation_advanced_multi_draw_buffers" supported="gl|gles2"/>
+        <extension name="GL_NVX_cross_process_interop" supported="gl|gles2">
+            <require comment="unpublished experimental extension">
```

Comment:
But the extension is being published now, at least in the sense that it will exist in the headers, right?

Reply:
not really. there is no spec being published, and it's definitely not the whole extension interface. 
Ideally it would be nice to keep this out of the header, but still keep the token associated with the extension? The only reason I'm adding it is so the CTS can pick up the definition of the token.

Reply:
If you put supported="disabled" (or any regexp that doesn't match an actual api tag like "gl" or "gles2", then the extension will not be included in output headers. Same mechanism as Vulkan. (Though you should check to make sure it works as intended here, since the scripts have diverged quite a lot).

Reply:
"disabled" does seem to work, and the CTS scripts are still happy with it as well.